flare_portal/experiments/templates/experiments/researcher_list.html

```diff
@@ -0,0 +1,90 @@
+                        <thead>
+                            <tr>
+                                <th></th>
+                                <th>User ID</th>
+                                <th>Delete</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            {% for researcher in researchers %}
+                                <tr>
+                                    <td><span class="text-muted">{{ forloop.counter }}</span></td>
+                                    <td>
+                                        <input type="text" name="userame" value="{{ researcher.username }}" maxlength="24" class="form-control" readonly>
+
+                                        {% for error in form.participant_id.errors %}
+                                            <div class="invalid-feedback">{{ error }}</div>
+                                        {% endfor %}
+                                    </td>
+                                    <td>
+                                        <a class="text-danger" href="{% url 'experiments:researcher_delete' project_pk=view.kwargs.project_pk researcher_pk=researcher.pk %}">
+                                            <i class="fe fe-trash-2"></i>
+                                        </a>
+                                        <!-- <form action="{% url 'experiments:researcher_delete' project_pk=view.kwargs.project_pk researcher_pk=researcher.pk %}" method="POST">
+                                            {% csrf_token %}
+                                            <input type="hidden" name="researcher_pk" value="{{ researcher.pk }}">
+                                            <button class="text-danger" style="-webkit-appearance: button;" type="submit">
+                                                <i class="fe fe-trash-2"></i>
+                                            </button>
+                                        </form> -->
+                                    </td>
+
+                                </tr>
+                            {% endfor %}
+
+                            <template x-if="showPicker == true">
+                                <tr>
+                                    <td><span class="text-muted" x-text="startIndex + 1"></span></td>
+                                    <td>
+                                        <form id="add-researcher-form" method="POST" action="">
+                                            {% csrf_token %}
+                                            {{ form.researchers|add_class:"form-control"|add_error_class:"is-invalid" }}
+                                        </form>
+                                    </td>
+
+                                    <td>
+                                        <a class="icon" href="#" @click.prevent="showPicker = false"><i class="fe fe-trash text-danger"></i></a>
```

Comment:
issue: I’m not sure what this does, but if there is `href="#"` it likely needs to be a `button` rather than `a`.

Reply:
This button removes the the box where you can add extra ID's.

We use anchor for ease of styling reasons, I'm hesitant to break the consistency since we're at the end and running out of time.

Reply:
👍  I think it’s ok for this to be deferred as long as we track the issue somewhere. I’m not sure what accessibility compliance targets KCL has – if this app and web UI fall under PSBAR, then this will likely need to at least be reported on the application’s accessibility statement.